yorkie/src/main/kotlin/dev/yorkie/document/Document.kt

```diff
@@ -129,7 +130,7 @@ public class Document(
     public fun updateAsync(
         message: String? = null,
         updater: suspend (root: JsonObject, presence: Presence) -> Unit,
-    ): Deferred<Boolean> {
+    ): Deferred<Result<Unit>> {
```

Comment:
typealias `OperationResult` can be applied here

Reply:
done [bb70e2e](https://github.com/yorkie-team/yorkie-android-sdk/pull/179/commits/bb70e2ed131470b26e9f69fce4560c936a7035ec)